backend/src/services/email/resend-config.service.ts

```
@@ -0,0 +1,296 @@
+        apiKeyEncrypted = EncryptionManager.encrypt(input.apiKey);
+      }
+
+      const result = await client.query(
+        `UPDATE email.resend_config SET
+           enabled = $1, api_key_encrypted = $2,
+           sender_email = COALESCE($3, sender_email),
+           sender_name = COALESCE($4, sender_name),
+           updated_at = NOW()
+         WHERE id = $5
+         RETURNING ${RESEND_CONFIG_COLUMNS}`,
+        [input.enabled, apiKeyEncrypted, input.senderEmail ?? null, input.senderName ?? null, existingRow.id]
```

ESLint

[error] Replace `input.enabled,·apiKeyEncrypted,·input.senderEmail·??·null,·input.senderName·??·null,·existingRow.id` with `⏎··········input.enabled,⏎··········apiKeyEncrypted,⏎··········input.senderEmail·??·null,⏎··········input.senderName·??·null,⏎··········existingRow.id,⏎········`

(prettier/prettier)